### DISCUSSION TITLE
How to I set the custom status?

### DISCUSSION THREAD

**Discussion by ghost** [Vedza/NitroSniperGo#240] [Q&A] [answered]
Hi, I was wondering how can I set the custom status? In features is that we can set a custom status, but how do I do it haha.

**Comment by Vedza** [comment, marked as answer]
Hi,
Here in the settings.
```json5
...

  },
  "status": {
    "main": "", // online, offline, idle, dnd, invisible
    "alts": "invisible" // online, offline, idle, dnd, invisible
  },

...
````
You can only change this status, not the game playing or others stuffs.

**Reply by ghost** [reply]
Yup, tysm. I'm just blind lmao
